mod/gcforums/views/default/forms/gcforums/edit.php

```diff
@@ -32,11 +31,17 @@
 ));
 
 
-// cyu - patched that only group owner/moderators/admin can do sticky topics
+// only group owner/moderators/admin can do sticky topics
 $gcf_current_user_guid = elgg_get_logged_in_user_guid();
 $gcf_moderator_users_guid = array();
 
-$gcf_moderator_user[] = get_entity($vars['group_guid'])->owner_guid;
+// not getting the group owner guid properly (use library in cp_notification)
+// check if the plugin cp_notification is active before using the function get_forum_in_group
+if (elgg_is_active_plugin('cp_notifications')) {
+	elgg_load_library('elgg:gc_notification:functions');
+	$gcf_moderator_user[] = get_entity(get_forum_in_group($object->getGUID(),$object->getGUID()))->owner_guid;
```

Comment:
This will get wiped out on L51, so maybe L45-53 need to be in and `else` branch of this conditional